src/olympia/addons/tests/test_views.py

```diff
@@ -963,6 +963,8 @@ def setUp(self):
         self.url = reverse_ns('addon-search')
         self.create_switch('return-to-amo', active=True)
         switch_is_active('return-to-amo')
+        self.create_switch('api-recommendations-priority', active=True)
+        switch_is_active('api-recommendations-priority')
```

Comment:
Did you forget an `assert`?

Reply:
no, it's just to check it so it's cached and the query doesn't show in when we're counting the number of db queries during ES related API calls.

Reply:
see the lines above